lib/bolt/cli.rb

```diff
@@ -474,7 +491,12 @@ def execute(options)
         when 'generate-types'
           code = generate_types
         when 'install'
-          code = install_puppetfile(config.puppetfile_config, config.puppetfile, config.modulepath.first)
+          assert_puppetfile_install_command(config.project.modules)
```

Comment:
Do we also want to show a deprecation warning for other puppetfile commands if modules is configured? Mostly it seems confusing that only one action is deprecated while others are valid. If I was a user I'd wonder if only `puppetfile install` was eventually getting deprecated, not the whole subcommand.

Reply:
The ticket called for just deprecating `puppetfile install`, but I agree that the entire `puppetfile` command makes more sense.